lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
+        this.analogPins = this.settings.analogPins || [ 14, 15, 16, 17, 18, 19 ];
```

Comment:
yes but I don't think this is a safe assumption. Probably better to throw an error and require the user to pass a correct set of analogPins